xml/depl_nodes.xml

```diff
@@ -2381,29 +2372,28 @@ getent group postgres | cut -d: -f3</screen>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>IP / Port for SMI-S</term>
+    <term><guimenu>IP</guimenu> / <guimenu>Port for SMI-S</guimenu></term>
     <listitem>
      <para>
       IP address and port of the ETERNUS SMI-S Server.
      </para>
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Username / Password for SMI-S</term>
+    <term><guimenu>Username</guimenu> / <guimenu>Password for SMI-S</guimenu></term>
```

Comment:
In the Web UI, the fields are named: 'IP for SMI-S', 'Port for SMI-S, 'Username for SMI-S', 'Password for SMI-S'